arch/arm64/boot/dts/xilinx/adi-ad9161-fmc-ebz.dtsi

```
@@ -0,0 +1,150 @@
+		jesd204-sysref-provider;
+
+		ad9508_0_c0:channel@0 {
+			reg = <0>;
+			adi,extended-name = "REF_CLK";
+			adi,driver-mode = <(DRIVER_PHASE_NORMAL | DRIVER_MODE_LVDS_1_00)>;
+			adi,divider-phase = <0>;
+			adi,channel-divider = <4>;
+		};
+
+		ad9508_0_c2:channel@2 {
+			reg = <2>;
+			adi,extended-name = "SYSREF2";
+			adi,driver-mode = <(DRIVER_PHASE_NORMAL | DRIVER_MODE_LVDS_1_00)>;
+			adi,divider-phase = <0>;
+			adi,channel-divider = <32>;
+		};
+
+		ad9508_0_c3:channel@3 {
+			reg = <3>;
+			adi,extended-name = "SYSREF";
+			adi,driver-mode = <(DRIVER_PHASE_NORMAL | DRIVER_MODE_LVDS_1_00)>;
+			adi,divider-phase = <0>;
+			adi,channel-divider = <32>;
+		};
+	};
+
+	dac0_ad9161: ad9161@1 {
+		#address-cells = <1>;
+		#size-cells = <0>;
+		compatible = "adi,ad9161";
+		reg = <1>;
+		spi-max-frequency = <1000000>;
+		clocks =  <&adf4355_clk 0>;
+		clock-names = "dac_clk";
+		spi-cpol;
+		spi-cpha;
+
+		adi,full-scale-current-mircoamp = <40000>;
+		dac_clk-clock-scales = <1 1>;
+		jesd204-device;
+		#jesd204-cells = <2>;
+		jesd204-top-device = <0>; /* This is the TOP device */
+		jesd204-link-ids = <0>;
+		jesd204-inputs = <&axi_ad9161_core 0 0>;
+
+		adi,jesd-subclass = <1>;
+		adi,dac-interpolation = <2>;
+		adi,channel-interpolation = <1>;
+		adi,clock-output-divider = <1>;
+		adi,syncoutb-signal-type-lvds-enable;
+		//adi,scrambling = <1>;
+		adi,sysref-mode = <2>; /* SYSREF_CONTINUOUS */
+
+		adi,subclass = <1>;			/* JESD SUBCLASS 0,1,2 */
+		adi,version = <1>;			/* JESD VERSION 0=204A,1=204B,2=204C */
+		adi,converters-per-device = <2>;	/* JESD M */
+		adi,octets-per-frame = <1>;		/* JESD F */
+		adi,frames-per-multiframe = <32>;	/* JESD K */
+		adi,converter-resolution = <16>;	/* JESD N */
+		adi,bits-per-sample = <16>;		/* JESD NP' */
+		adi,control-bits-per-sample = <0>;	/* JESD CS */
+		adi,lanes-per-device = <8>;		/* JESD L */
+		adi,samples-per-converter-per-frame = <2>; /* JESD S */
+		adi,interpolation = <2>;
+		jesd204-ignore-errors;
```

Do we really need to ignore errors?